package.json

```diff
@@ -140,6 +140,7 @@
     "@babel/preset-react": "^7.9.1",
```

Comment:
We'll want to update the script commands to use the new cli and webpack dev server to prepare for removing terra-toolkit. They will look something like this, copying from a different project:

```
    "start": "webpack-dev-server",
    "start:prod": "webpack-dev-server --env.disableHotReloading -p",
    "start-static": "npm run pack && terra express-server --site ./build",
 ```

Reply:
updated here https://github.com/cerner/terra-application/pull/129/commits/dc447b795056de76542c01bf94d0e810e1d9814c